ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -404,6 +407,642 @@ objects:
+        commonName              = "CI Proxy"
+        [ req_ext ]
+        subjectAltName          = "DNS.1:*.compute-1.amazonaws.com"
+        [ v3_ca ]
+        subjectKeyIdentifier = hash
+        authorityKeyIdentifier = keyid:always,issuer
+        basicConstraints = critical, CA:true
+        keyUsage = critical, digitalSignature, cRLSign, keyCertSign
+        [ v3_intermediate_ca ]
+        subjectKeyIdentifier = hash
+        authorityKeyIdentifier = keyid:always,issuer
+        basicConstraints = critical, CA:true, pathlen:0
+        keyUsage = critical, digitalSignature, cRLSign, keyCertSign
+        [ usr_cert ]
+        basicConstraints = CA:FALSE
+        nsCertType = client, email
+        nsComment = "OpenSSL Generated Client Certificate"
+        subjectKeyIdentifier = hash
+        authorityKeyIdentifier = keyid,issuer
+        keyUsage = critical, nonRepudiation, digitalSignature, keyEncipherment
+        extendedKeyUsage = clientAuth, emailProtection
+        [ server_cert ]
+        basicConstraints = CA:FALSE
+        nsCertType = server
+        nsComment = "OpenSSL Generated Server Certificate"
+        subjectKeyIdentifier = hash
+        authorityKeyIdentifier = keyid,issuer:always
+        keyUsage = critical, digitalSignature, keyEncipherment
+        extendedKeyUsage = serverAuth
+        [ crl_ext ]
+        authorityKeyIdentifier=keyid:always
+        [ ocsp ]
+        basicConstraints = CA:FALSE
+        subjectKeyIdentifier = hash
+        authorityKeyIdentifier = keyid,issuer
+        keyUsage = critical, digitalSignature
+        extendedKeyUsage = critical, OCSPSigning
+        EOF
+
+          popd
+          uuidgen | sha256sum | cut -b -32 > intpassfile
+
+          openssl genrsa -aes256 \
+              -out ${INTERMEDIATE}/private/intermediate.key.pem \
+              -passout file:intpassfile 4096 2>/dev/null
+
+          chmod 400 ${INTERMEDIATE}/private/intermediate.key.pem
+
+          openssl req -config ${INTERMEDIATE}/openssl.cnf -new -sha256 \
+              -key ${INTERMEDIATE}/private/intermediate.key.pem \
+              -passin file:intpassfile \
+              -out ${INTERMEDIATE}/csr/intermediate.csr.pem 2>/dev/null
+
+          openssl ca -config openssl.cnf -extensions v3_intermediate_ca \
+              -days 3650 -notext -md sha256 \
+              -batch \
+              -in ${INTERMEDIATE}/csr/intermediate.csr.pem \
+              -passin file:capassfile \
+              -out ${INTERMEDIATE}/certs/intermediate.cert.pem 2>/dev/null
+
+          chmod 444 ${INTERMEDIATE}/certs/intermediate.cert.pem
+
+          openssl verify -CAfile certs/ca.cert.pem \
+              ${INTERMEDIATE}/certs/intermediate.cert.pem
+
+          cat ${INTERMEDIATE}/certs/intermediate.cert.pem \
+              certs/ca.cert.pem > ${INTERMEDIATE}/certs/ca-chain.cert.pem
+
+          chmod 444 ${INTERMEDIATE}/certs/ca-chain.cert.pem
+          popd
+        }
+
+        function generateProxyIgnition() {
```

Comment:
nit: You have some irregular function naming:

```console
$ git show origin/pr/4719 -- ci-operator/templates/openshift/installer/ | grep function
+        function generate_proxy_certs() {
+        function generateProxyIgnition() {
+        function generateProxyTemplate() {
+        function printLogs() {
```

Looks like the templates are currently split between hyphen names and underscore names, but I don't see any camelCase.  Can you pick hypens or underscores for this PR and apply them to all your new functions?

Reply:
@wking do you feel the same way about `printLogs` ? that is used as part of the `proxy.sh` in the proxy image...

Reply:
> do you feel the same way about `printLogs`?
I don't see why we'd have different conventions for different sections within openshift/release.  Do you have a reason?

Reply:
not really.. i'll update it to use underscores